docs/dev/table/hive/read_write_hive.md

```diff
@@ -110,6 +110,13 @@ __________ __________
 
 {% endhighlight %}
 
+### Querying Hive views
+
+If you need to query Hive views, please note:
+
+1. You have to switch to the Hive catalog before you can query views in that catalog.
```

Comment:
What do you mean?

Reply:
Users need to make sure Hive catalog is the current catalog, otherwise the underlying tables cannot be resolved.

Reply:
maybe just rephrase as following?

```suggestion
1. You have to use the Hive catalog as your current catalog before you can query views in that catalog. It can be done by either `tableEnv.useCatalog(...)` in Table API or `USE CATALOG ...` in SQL Client
```

Reply:
Of cause we need hive catalog or use `catalog.database.table`. I think this sentence is a little redundant...

Reply:
I think you should add a reason to tell reader why...